src/stp/new_plays/DefendPass.cpp

```diff
@@ -35,7 +35,7 @@ DefendPass::DefendPass() : Play() {
         std::make_unique<role::Formation>(role::Formation("offender_2"))};
 }
 
-uint8_t DefendPass::score(world_new::World *world) noexcept { return 90; }
+uint8_t DefendPass::score(world_new::World *world) noexcept { return 100; }
```

Comment:
Again, make a constant for this class variable.

Reply:
No because the scoring will be fixed in  a future PR